src/CodeMirrorBlocks.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import ToggleEditor from './ui/ToggleEditor';
+import wescheme from '../src/languages/wescheme';
+
+// Consumes a DOM node to host the editor, a language object and the code
+// to render. Produces an object-representation of CMB, allowing for
+// integration with external (non-react) code
+export class CodeMirrorBlocks {
+  constructor(dom, language = wescheme, code = "") {
+    let obj = {};
+    ReactDOM.render(
+      <ToggleEditor language={language} initialCode={code} external={obj} />,
+      dom
+    );
+    return obj;
+  }
+}
```

Comment:
I'm confused that this file shows up in the diff, because it's already in `new-reactify`. And in fact, this file is somewhat out of date!

Reply:
OH! If it's already in there, the bulk of this PR is unneeded. In that case, let me close the PR and let Raymond work of `new-reactify`.